packages/webpack/css-extract-webpack-plugin/runtime/hotModuleReplacement.cjs

```diff
@@ -17,29 +17,38 @@ function debounce(fn, time) {
 }
 
 function updateStyle(cssId = 0) {
-  const filename = __webpack_require__.lynxCssFileName;
-  if (!filename) {
+  const cssHotUpdateList = __webpack_require__.lynxCssFileName;
+  if (!cssHotUpdateList) {
     throw new Error('Css Filename not found');
```

Comment:
The same for error msg

Reply:
Regarding error msg, how about `CSS file ${filename} is not found`?